provider/testdata/benchmarks/storage-bucket-bench.txt

```diff
@@ -0,0 +1,70 @@
+Benchmark/storage-bucket/preview-initial/ResourceProvider/CheckConfig/pulumi:providers:gcp                                           1         227827208 ns/op
+Benchmark/storage-bucket/preview-initial/ResourceMonitor/RegisterResource/pulumi:providers:gcp                                       1         439078125 ns/op
+Benchmark/storage-bucket/preview-initial/ResourceProvider/Check/gcp:storage/bucket:Bucket                                            1           2120625 ns/op
+Benchmark/storage-bucket/preview-initial/ResourceProvider/Create/gcp:storage/bucket:Bucket                                           1           5296000 ns/op
+Benchmark/storage-bucket/preview-initial/ResourceMonitor/RegisterResource/gcp:storage/bucket:Bucket                                  1         147890542 ns/op
+Benchmark/newPlugin/yaml:/Users/mjeffryes/workspace/pulumi/gcp/.pulumi/bin/pulumi-language-yaml                                      1          16789166 ns/op
+Benchmark/newPlugin/gcp                                                                                                              1         213277958 ns/op
+Benchmark/storage-bucket/preview-initial/ResourceProvider/CheckConfig/pulumi:providers:gcp                                           1         250674333 ns/op
+Benchmark/storage-bucket/preview-initial/ResourceMonitor/RegisterResource/pulumi:providers:gcp                                       1         465126792 ns/op
+Benchmark/storage-bucket/preview-initial/ResourceProvider/Check/gcp:storage/bucket:Bucket                                            1           1320958 ns/op
+Benchmark/storage-bucket/preview-initial/ResourceProvider/Create/gcp:storage/bucket:Bucket                                           1           5262666 ns/op
+Benchmark/storage-bucket/preview-initial/ResourceMonitor/RegisterResource/gcp:storage/bucket:Bucket                                  1         155507333 ns/op
+Benchmark/newPlugin/yaml:/Users/mjeffryes/workspace/pulumi/gcp/.pulumi/bin/pulumi-language-yaml                                      1          16394458 ns/op
+Benchmark/newPlugin/gcp                                                                                                              1         211621667 ns/op
+Benchmark/storage-bucket/preview-initial/ResourceProvider/CheckConfig/pulumi:providers:gcp                                           1         238482792 ns/op
+Benchmark/storage-bucket/preview-initial/ResourceMonitor/RegisterResource/pulumi:providers:gcp                                       1         451132708 ns/op
+Benchmark/storage-bucket/preview-initial/ResourceProvider/Check/gcp:storage/bucket:Bucket                                            1           1878083 ns/op
+Benchmark/storage-bucket/preview-initial/ResourceProvider/Create/gcp:storage/bucket:Bucket                                           1           4107000 ns/op
```

Comment:
Ah very interesting so the idea is to check in baselines? I think I like that. We have a similar problem with upgrade test recording baselines and the question of whether these belong in the repo still remains. We could consider alternate storage. Perhaps GitHub actions permits storing long-lived artifacts, or if that fails we can use an S3 bucket. But certainly not blocking on this, a great start.